pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -691,17 +691,19 @@ func (k *Kubernetes) Deploy(komposeObject kobject.KomposeObject, opt kobject.Con
 }
 
 // Undeploy deletes deployed objects from Kubernetes cluster
-func (k *Kubernetes) Undeploy(komposeObject kobject.KomposeObject, opt kobject.ConvertOptions) error {
+func (k *Kubernetes) Undeploy(komposeObject kobject.KomposeObject, opt kobject.ConvertOptions) []error {
+	var ErrorList []error
```

Comment:
There shouldn't be capital 'E'. it should be lower-case.
Uppercase in go means public variable, we don't want that here.